data/translations/pt_PT.ts

```diff
@@ -2,6 +2,20 @@
 <!DOCTYPE TS>
 <TS version="2.1" language="pt_PT">
 <context>
+    <name>PictureBox</name>
+    <message>
+        <source>Press to login</source>
+        <translation type="unfinished"></translation>
```

Comment:
seems like it just adds a new (untranslated) string to all translation files so nothing tangible will happen. probably best if translators will rebase their translations separately?